package.json

```diff
@@ -10,7 +10,7 @@
     "build": "npm run build --workspaces --if-present",
     "check:pinned-deps": "tsx tools/ensure-pinned-deps",
     "check": "npm run check --workspaces --if-present && run-p check:*",
-    "clean": "npm run clean --workspaces --if-present && rimraf **/.wireit",
+    "clean": "npm list rimraf && npm run clean --workspaces --if-present && rimraf **/.wireit",
```

Comment:
so if the rimraf is not yet installed, then nothing will be cleaned up? The behaviour looks unexpected and I think it's better to error in that case (I think that would be the current behaviour)

Reply:
Yes, now it will throw an error. Alright, I will think about different solution